docs/UserGuide.md

```diff
@@ -3,52 +3,35 @@ layout: page
 
-* What food items are about to expire?
-* How much of each condiment do I have?
-* Which food items are newly purchases?
-* ...and many more!
+Using a Command-Line interface, efficiently interact with FoodRem without needing to even touch your mouse! With a focus on user-friendliness, FoodRem is easy to learn! 
 
-With a few quick commands, you can have complete control of your perishable goods, so you can focus on what is important: serving your customers, improving menu recipes, and transforming your business into the next success story.
-
-## Key Features
-
-1. Add, update and delete inventory items
-1. Search and sort food items by:
-   * Name
-   * Quantity
-   * Bought date
-   * Expiry date
-   * Tags
-1. Tag items to group them into categories
-
-## Purpose, Scope and Audience
-
-### Purpose
-
-FoodRem helps small businesses to easily manage consumables and perishable goods within a single inventory to reduce time spent on managing and restocking inventory.
-
-### Scope
-
-This app is targeted at small F&B businesses which may struggle in inventory management due to a lack of streamlined process and manpower.
-
-### Audience
-
-This guide is targeted at users using FoodRem who wants to find out more about the different commands to manage their inventory in a quicker and more efficient manner.
-
-Readers do not have to be familiar with the command line to use this guide effectively. The only expectation we have of you is to carefully read through the different sections.
+This User Guide provides an in-depth documentation to help integrate FoodRem into your operational workflows. It covers how to launch FoodRem, core FoodRem features and commands, common terms and definitions used in FoodRem, and some troubleshooting recommendations. In addition, a quick start guide is available that gives a whirlwind tour to help get you started. 
 
 {% include page-break.html %} <!-- Force a page break -->
 
 {% include toc.md header=true show-in-toc=true ordered=true %}
 
-## How to use the User Guide
+## Features 
+There are two core features that FoodRem provides: The Inventory Management System, and inventory analysis capabilities. 
+
+TODO: Add links to e.g. name, bought date, expiry date, etc.
+### Inventory Management System
+ FoodRem allows you to track your inventory during your daily operations. 
+ 1. Quickly **add, remove, and view** your available inventory. 
```

Comment:
I feel it will be better to phrase is as 

Quickly **create, view, edit and remove** items in your inventory.